server/index.js

```diff
@@ -22,17 +22,17 @@ app.get('/app/:gameid', (req, res) => {
 
 router.get('/gamereviews/:gameid', async (req, res) => {
   let { gameid } = req.params;
-  if (parseInt(gameid) <= 0 || parseInt(gameid) > 100) {
+  console.log(req.originalUrl)
+  if (parseInt(gameid) <= 0 || parseInt(gameid) > 30000) {
```

Comment:
This part is a little confusing - I thought we were supposed to be supporting 10m primary records. Does this mean the API doesn't support greater than 30,000?

Reply:
As for Reviews service, I believe that reviews would be my primary data, for each game, there could be an x amount of reviews. So my service doesn't necessarily require 10M games , but 10M 'reviews' distributed amongst x amount of games. I don't think/know if there is a service that will provide me the game ids or information about particular game ids. If anything, this can easily be changed to be up to 10million different games but again the main concern for my service is about reviews and reviews are my primary data, not games